integration_test_lite.py

```diff
@@ -23,6 +23,7 @@ def __init__(self):
         self.access_token = None
         self.total = self.config_json['request_timeout']
         self.timeout = aiohttp.ClientTimeout(total=self.total)
+        self.good_apis = []
```

Comment:
I think you should do something similar as asynchronous `get_bad_apis` to get good apis. In fact, I believe you can modify the code there so that you don't need to rewrite again by yourself.

Reply:
I refactored `bad_response()` to have it parse through both failing and passing tests, so no need for another function. 'bad_response()' was already kind of a misnomer, as it was already parsing through all responses